alwaysontop/index.js

```diff
@@ -1,5 +1,5 @@
 const setupAlwaysOnTopRender = require('./render');
-const { attachAlwaysOnTopToWindow, setupAlwaysOnTopMain } = require('./main');
```

Comment:
This was only used in the intermediate undocked implementation, right? Else, we'd have to bump the major since it's an API breakage.

Reply:
that's correct